Flight/FlightManagement.cs

```diff
@@ -0,0 +1,275 @@
+﻿using AirportTicketBookingSystem.Enums;
+using AirportTicketBookingSystem.FileSystem;
+using AirportTicketBookingSystem.Exceptions;
+using System.Linq;
+
+namespace AirportTicketBookingSystem.Flights
+{
+    public class FlightManagement
+    {
+        public FlightManagement()
+        {
+            Flights = new List<Flight>();
+            PassengerFlights = new List<Flight>();
+            AddDefaultFlight();
+        }
+
+        public List<Flight> Flights { get; set; }
+        public List<Flight> PassengerFlights { get; set; }
+
+        public async Task BatchFlightUpload()
```

Comment:
It looks like you don't implement the exact requirement from this method, the Task needs to depend on the file system as storage layer, so the Idea is we need a file with all flights, and they should be loaded into Flights, so if I'm a manger I need to add batch/bulk of flights instead of add one by one I will get a new flight file and upload so add the new flights from my file to the origin file, then the Flights list also.